packages/graphql-cli-graphback-plugin/tsconfig.json

```diff
@@ -0,0 +1,36 @@
+{ 
+    "include": [
+      "./src/**/*.ts"
+    ],
+    "compilerOptions": {
```

Comment:
See other packages that do `  "extends": "../../tsconfig.json",` those were copied as this was separate package.